### DISCUSSION TITLE
Is it normal for pnpm to crash when there's an error in the content script?

### DISCUSSION THREAD

**Discussion by maribox** [wxt-dev/wxt#1295] [Q&A] [answered]
Hey!
I have a problem when I run `pnpm dev` where, when I edit it auto-saves and tries to execute the content script, then the command crashes, so I have to run pnpm dev again , opening a new browser window which is really annoying. Is this to be expected? It crashes with a "BABEL_PARSER_SYNTAX_ERROR". I use svelte and wxt 0.19.13.

**Comment by aklinker1** [comment]
Not intentional, this is a bug! So it happens when you save a file used by a content script and it has a syntax error?

I've noticed it crashes every once in a while, but was never able to pinpoint the exact reason.

**Reply by maribox** [reply]
Yes exactly. Should I make an issue?

**Reply by aklinker1** [reply, marked as answer]
Please do!
